packages/foreman/foreman-release/foreman-client.repo

```diff
@@ -3,11 +3,11 @@ name=Foreman client nightly
 baseurl=https://yum.theforeman.org/client/nightly/$DIST/$basearch
 enabled=1
 gpgcheck=0
-gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-foreman
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-foreman-client
```

Comment:
where is this file coming from? I don't see it in `%files`

Reply:
The repo is currently not signed (hence the `gpgcheck=0`). I just thought I should rename this to make it clear it's a different file. Users have been setting gpgcheck=1.

Reply:
oooh, I'll be damned for not seeing the `gpgcheck=0` above.

Reply:
See https://projects.theforeman.org/issues/26278 as well. I also consider leaving the line out.